### DISCUSSION TITLE
Where can I find config file of alpine-vaultwarden container?

### DISCUSSION THREAD

**Discussion by emanueletocci** [tteck/Proxmox#1512] [Q&A] [answered]
Hi, I'm looking for config file of alpine-vaultwarden container but I'm not able to find it.... I saw into /var/lib/vaultwarden but there isn't .env files or .json... Where is it?
![immagine](https://github.com/tteck/Proxmox/assets/83188429/527ac535-0682-41d6-b7ae-2d7e14de18ed)

I would enable admin page and replace database with an older one (to restore passwords)...

**Comment by tteck** [comment, marked as answer]
`nano /etc/conf.d/vaultwarden`

**Reply by emanueletocci** [reply]
Thank you so much... awesome work, congratulation!🔥

**Reply by tteck** [reply]
![Screenshot 2023-06-19 2 40 28 PM](https://github.com/tteck/Proxmox/assets/21342670/60f4f283-123d-4036-a0e6-34a3c13e6fac)
